parm/use_cases/met_tool_wrapper/EnsembleStat/EnsembleStat_python_embedding.conf

```
@@ -114,20 +111,13 @@ ENSEMBLE_STAT_OUTPUT_FLAG_ORANK = BOTH
 ENSEMBLE_STAT_OUTPUT_FLAG_SSVAR = BOTH
 ENSEMBLE_STAT_OUTPUT_FLAG_RELP = BOTH
 
-ENSEMBLE_STAT_ENSEMBLE_FLAG_LATLON = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_MEAN = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_STDEV = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_MINUS = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_PLUS = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_MIN = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_MAX = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_RANGE = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_VLD_COUNT = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_FREQUENCY = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_NEP = FALSE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_NMEP = FALSE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_RANK = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_WEIGHT = FALSE
+ENSEMBLE_STAT_NC_ORANK_FLAG_LATLON = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_MEAN = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_RAW = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_RANK = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_PIT = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_VLD_COUNT = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_WEIGHT = TRUE
```

```suggestion
ENSEMBLE_STAT_NC_ORANK_FLAG_WEIGHT = FALSE
```
Same comment as previous about changing weight from FALSE to TRUE.